src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
-body {
+.container {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__hand--minute {
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    top: calc(50% - 20vmin);
+    animation: rotate-minute 3600s steps(60) infinite;
+    transform-origin: center bottom;
+  }
+
+  &__hand--second {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    top: calc(50% - 38vmin);
+    animation: rotate-second 60s linear infinite;
+    transform-origin: center bottom;
+  }
+
+  &__center {
+    position: absolute;
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 50%;
+    background-color: #f6a603;
+    z-index: 1;
+  }
+}
+
+.stopwatch--speed-up .stopwatch__hand--minute {
+  animation: rotate-minute 600s step(60) infinite;
+}
+
+.stopwatch--speed-up .stopwatch__hand--second {
+  animation: rotate-minute 10s linear infinite;
```

Comment:
There seems to be a typo in the animation property. It currently uses 'rotate-minute', but it should likely be 'rotate-second' to match the intended animation for the second hand.